mining/worker.go

```diff
@@ -253,6 +255,22 @@ func createProof(challengeSeed proofs.PoStChallengeSeed, createPoST DoSomeWorkFu
 	return c
 }
 
+// CreateTicket computes a valid ticket using the supplied proof
+// []byte and the minerAddress address.Address.
+//    returns:  []byte -- the ticket.
+func CreateTicket(proof proofs.PoStProof, signerAddr address.Address, signer types.Signer) []byte {
+	buf := append(proof[:], signerAddr.Bytes()...)
+	h := blake2b.Sum256(buf)
+	//return h[:]
+
+	ticket, err := signer.SignBytes(h[:], signerAddr)
+	if err != nil {
+		errMsg := fmt.Sprintf("SignBytes error in CreateTicket: %s", err.Error())
+		panic(errMsg)
```

Comment:
Seems panic worthy to me.  Would be good to check with our local panic enforcer (@phritz) who has many times told me to stop panicing when at the time it was always similarly obvious to me.